WordPress/src/main/java/org/wordpress/android/ui/news/LocalNewsService.kt

```diff
@@ -0,0 +1,39 @@
+package org.wordpress.android.ui.news
+
+import android.arch.lifecycle.MutableLiveData
+import android.content.Context
+import org.wordpress.android.models.news.NewsItem
+import org.wordpress.android.models.news.NewsItemType
+import javax.inject.Inject
+
+/**
+ * Service for fetching data for a News Card (Card with a new feature/update announcement) from a local resource.
+ *
+ * This is just a temporary solution until News Cards are supported on the server.
+ */
+class LocalNewsService @Inject constructor(private val context: Context) : NewsService {
+    val data: MutableLiveData<NewsItem?> = MutableLiveData()
+    var newsItem: NewsItem? = null
+
+    override fun getNewsItem(): MutableLiveData<NewsItem?> {
+        if (newsItem == null) {
+            newsItem = loadCardFromResources()
+            data.value = newsItem
```

Comment:
this is ok at the moment but it gives the observer a bit different behaviour than I expect we'll have once this feature is implemented. I assume at one point you'll start using `postValue` instead with a value coming from the backend. The `LiveData` will start empty and will get the value lazily. Right now this behaves like a field getter (but as I said, at the moment it doesn't matter, it's just something worth considering).

Reply:
Yep, I agree this implementation works for static local resource only. 

The `RemoteNewsService` will need to listen for api request result events and set the value when an event is received. 

Thanks for the suggestion, but If you don't mind, I wouldn't worry about it now.